Aquarius/Src/Aquarius/Core/Window.cpp

```diff
@@ -1,10 +1,10 @@
 #include "Window.h"
 #include "Log.h"
-#include <assert.h>
 #include "Aquarius/Events/Keyboard.h"
 #include "Aquarius/Events/Mouse.h"
 #include "Aquarius/Core/Input.h"
```

Comment:
Shouldn't the core stuff be above all this?

Reply:
A good example here of what I mean by the two styles of includes for the same subdirectory here.

Reply:
@liamjwarren Just wondering why you say that, because its local or because its alphabetical? I think we are gonna go with alphabetical for this